library/build.gradle

```diff
@@ -50,6 +50,8 @@ dependencies {
 
     testCompile 'junit:junit:4.12'
     testCompile 'org.mockito:mockito-core:1.10.19'
+    testCompile 'org.powermock:powermock-module-junit4:1.6.4'
```

Comment:
:-1: powermock is not cool, I assume you've added this to either mock static methods or override final classes? A better way is to highlight where these design issues are in the code so that we can make the code better rather than using a brute force tool to get around the issue